src/sentry/runner/commands/configoptions.py

```diff
@@ -57,7 +58,11 @@ def _attempt_update(
 
     if not dry_run:
         options.set(key, value, coerce=False, channel=options.UpdateChannel.AUTOMATOR)
-    click.echo(UPDATE_MSG % key)
+    if last_update_channel is not None:
+        click.echo(UPDATE_MSG % key)
+        click.echo(safe_dump(db_value))
```

Comment:
python repr will be more useful than yaml, imo

P4: not a blocking issue

Reply:
Fair, will change. I picked this from the drift case. 
In that case I wanted to show the yaml as the expected flow is to fix the drift in the configmap. There having the yaml means you can copy paste the output of the script into the configmap without changing the format.